experiments/heroku/README.md

```diff
@@ -0,0 +1,47 @@
+# Heroku Deployment for cell×gene
+
+## Quick Start
+
+Clicking on the following button will forward you to heroku to begin the deployment process
+
+<a href="https://heroku.com/deploy?template=https://github.com/chanzuckerberg/cellxgene/tree/heroku">
+  <img src="https://www.herokucdn.com/deploy/button.svg" alt="Deploy">
+</a>
+
+If not already logged in to heroku, there you will be prompted to login or sign up for an account.
+
+Once logged in you will be sent to the setup page. Here you can set some of the basic settings for the app:
+
+#### Default Settngs
+
+- `App name`: the unique name name for your deployment.
+  - This will also serve as the default url (e.g. https://cellxgene.herokapp.com/)
+- `App owner`: Who will own this app. Either you personally or an organiziation/team
+- `Region`: Location of server where app will be deployed (EU or US)
+
+#### Config Vars
+
+- `DATASET`: A _publicly_ accessable URL pointing to a .h5ad file to view
+  - This is defaulted to pbm3k.h5ad
+
+After filling out the settings and pressing the `Deploy app` button heroku will begin building your deployment. This process will take a few minutes, but once completed you will have a personal free hosted version of cell×gene!
+
+## What Is Heroku and Why Should I Use It?
```

Comment:
Thanks for putting this together! This is a great start. Keeping in mind that web development / deployment is not something that many computational biologists have any experience with, I think there are some opportunities to expand.

Suggested revisions:
Section on "What is Heroku?":
1 - Please remove the quote and expand the summary of Heroku for the layperson ("Heroku is a service offered by Amazon to host applications on the web...")
2 - Make it clear why cellxgene needs something like heroku ("Cellxgene currently can't be hosted as a static web page because...."); include the idea that it needs a container.
3 - Make it clear that Heroku is one of several available options for hosting an application like cellxgene. 

Section on "When is Heroku a good option?"
1 - Current text is generally great. 
2 - Please revise this sentence:
"If you want to quickly deploy an instance of cellxgene and, if not using a small dataset, okay with paying for a tiered dyno, this is a viable solution." --> 
"This is a good option if you want to quickly deploy an instance of cellxgene to the web. Heroku deployments are free for small datasets up to XX size. See below regarding larger datasets."

Reply:
👍